us3_procedures.sql

```diff
@@ -324,12 +403,18 @@ BEGIN
       SET @personGUID = NULL;
       SET @USERLEVEL  = NULL;
 
+      RETURN( @US3_LAST_ERRNO );
     ELSEIF ( activated = false ) THEN
       SET @US3_LAST_ERRNO = @INACTIVE;
-      SET @US3_LAST_ERROR = CONCAT( 'MySQL: This account has not been activated yet. ',
-                                    'Please activate your account first. ',
-                                    'The activation code was sent to your e-mail address: ',
-                                     p_email);
+      IF ( @authenticatePAM != 1 ) THEN
+         SET @US3_LAST_ERROR = CONCAT( 'MySQL: This account has not been activated yet. ',
+                                       'Please activate your account first. ',
+                                       'The activation code was sent to your e-mail address: ',
```

Comment:
In this file we use "e-mail" address in the message, but in checkuser.php the message is "email". Which is the correct spelling throughout the project?

Reply:
> In this file we use "e-mail" address in the message, but in checkuser.php the message is "email". Which is the correct spelling throughout the project?

I don't think we have a consistent usage. If we should do so, create a new issue as everywhere it is used should be checked (gui, php code in various repos).